src/components/widgets/WaterTemperatureWidget.tsx

```diff
@@ -0,0 +1,38 @@
+import { useNavigation } from '@react-navigation/core';
+import React, { useCallback } from 'react';
+
+import { Icon, normalize, texts } from '../../config';
+import { useHomeRefresh } from '../../hooks';
+import { useWaterTemperature } from '../../hooks/waterTemperature';
+import { QUERY_TYPES } from '../../queries';
+import { ScreenName, WidgetProps } from '../../types';
+
+import { DefaultWidget } from './DefaultWidget';
+
+export const WaterTemperatureWidget = ({ text, additionalProps }: WidgetProps) => {
+  const navigation = useNavigation();
+  const { temperature, refresh } = useWaterTemperature();
+
+  const onPress = useCallback(() => {
+    navigation.navigate(ScreenName.Html, {
+      title: text ?? texts.waterTemperature.headerTitle,
+      query: QUERY_TYPES.WATER_TEMPERATURE,
+      queryVariables: { name: additionalProps?.staticContentName ?? 'water-temperature' }
+    });
+  }, [navigation, text]);
+
+  useHomeRefresh(refresh);
+
+  if (!temperature) return null;
```

Comment:
i checked locally and it is confusing, that this widget is the only one loading later then the others. let's not `null` it here but use a fallback `'—'` like for the weather widget

<img width="234" alt="image" src="https://user-images.githubusercontent.com/1942953/189367828-57722af5-3a27-4bb1-8d0c-30b1ce6be200.png">

```
count={(temperature ?? '—') + '°C'}
```

Reply:
I can change this. 👍